README.old.md

```diff
@@ -1 +1,94 @@
-# math-magicians
\ No newline at end of file
+# math-magicians
+
+Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+# Screenshot
+
+![foodWebsiteModule2Capstone](https://user-images.githubusercontent.com/69027469/169603706-d24a9c5b-3ea1-4871-b5c7-aea702ac3991.jpeg)
+
+## Built With
+
+- HTML
+
+- CSS
+
+- JavaScript
+
+- React JS
+
+- Webpack
+
+- Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+
+- Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+- ESlint (A mighty, modern linter that helps you avoid errors and enforce conventions in JavaScript codes)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link `git@github.com:EidHachem/math-magicians.git`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `git clone git@github.com:EidHachem/math-magicians.git`.
+- go to the repository folder in your command prompt `cd math-magicians`
+- use `npm start` to start live server
```

Comment:
- [ ] (OPTIONAL) You should place the command `npm install` before `npm start` since you can't start the server without installing the dependencies first 👍

Reply:
@Kingstalux  Done.
Thank you again 👍🏽